data/json/mapgen/small_town_buildings/lumberyard_1_1.json

```diff
@@ -0,0 +1,98 @@
+        "................|.....|g",
+        "....#######...........|g",
+        "......................|g",
+        "......................|g",
+        "......................|g",
+        "......................|g",
+        "....||..|||||||+|.....|g",
+        "....|...b|#&#&..|.....|g",
+        "....|{..b|......|b....|g",
+        "....|#...|+|+|..|b....|g",
+        "....|####|t|t|T.|.....|g",
+        "|||||||||||||||||||++||g",
+        "__________________r___-g",
+        "__________________r___-g",
+        "______________________-g",
+        "______________________-g",
+        "______________________-g",
+        "______________________-g",
+        "__________________O_X_-g",
+        "______________________-g",
+        "______________________-g",
+        "-----------------------g",
+        "gggggggggggggggggggggggg"
+      ],
+      "terrain": {
+        "#": "t_floor",
+        "&": "t_floor",
+        "+": "t_door_locked_interior",
+        "-": "t_chainfence_v",
+        ".": "t_floor",
+        "O": "t_pavement",
+        "T": "t_floor",
+        "X": "t_pavement",
+        "_": "t_pavement",
+        "b": "t_floor",
+        "g": "t_grass",
+        "r": "t_railing_v",
+        "t": "t_floor",
+        "{": "t_floor",
+        "|": "t_wall"
+      },
+      "furniture": {
+        "#": "f_counter",
+        "&": "f_sink",
+        "O": "f_crate_o",
+        "T": "f_trashcan",
+        "X": "f_crate_c",
+        "b": "f_bench",
+        "t": "f_toilet",
+        "{": "f_fridge"
+      },
+      "toilets": {
+        "t": {}
+      },
+      "place_items": [
+        { "item": "snacks", "x": 6, "y": 11, "chance": 30 },
+        { "item": "fridge", "x": 5, "y": 9, "chance": 40 },
+        { "item": "cannedfood", "x": 7, "y": 11, "chance": 35 },
+        { "item": "trash", "x": 14, "y": 11, "chance": 50 }
+      ],
+      "add": [
+        { "item": "2x4", "x": 5, "y": 15 },
+        { "item": "2x4", "x": 8, "y": 15 },
+        { "item": "2x4", "x": 10, "y": 15 },
+        { "item": "2x4", "x": 11, "y": 15 },
+        { "item": "2x4", "x": 12, "y": 15 },
+        { "item": "2x4", "x": 13, "y": 15 },
+        { "item": "2x4", "x": 14, "y": 15 },
+        { "item": "2x4", "x": 15, "y": 15 },
+        { "item": "2x4", "x": 5, "y": 17 },
+        { "item": "2x4", "x": 7, "y": 17 },
+        { "item": "2x4", "x": 8, "y": 17 },
+        { "item": "2x4", "x": 9, "y": 17 },
+        { "item": "2x4", "x": 10, "y": 17 },
+        { "item": "2x4", "x": 11, "y": 17 },
+        { "item": "2x4", "x": 12, "y": 17 },
+        { "item": "2x4", "x": 14, "y": 17 },
+        { "item": "2x4", "x": 5, "y": 19 },
+        { "item": "2x4", "x": 6, "y": 19 },
+        { "item": "2x4", "x": 7, "y": 19 },
+        { "item": "2x4", "x": 9, "y": 19 },
+        { "item": "2x4", "x": 12, "y": 19 },
+        { "item": "2x4", "x": 13, "y": 19 },
+        { "item": "2x4", "x": 14, "y": 19 },
+        { "item": "2x4", "x": 15, "y": 19 }
```

Comment:
A ton of hard-set planks will look bad. Those should be spawned over an area with rolled x, y and repeat.

Reply:
Fixed this.